interfaces/cython/SConscript

```diff
@@ -76,11 +76,109 @@ mod = build(localenv.Command('#build/python/cantera/__init__.py', 'setup.py',
+
+solution_properties = ""
+getter_only_properties_template = Template("""
+    @property
+    def ${name}(self):
+        return Q_(self._phase.${name}, ${units})
+    """)
+
+getter_only_properties = ["P","P_sat","T","T_sat","atomic_weight","chemical_potentials","concentrations","critical_pressure","critical_temperature","electric_potential","electrochemical_potentials","isothermal_compressibility","max_temp","mean_molecular_weight","min_temp","molecular_weights","partial_molar_cp","partial_molar_enthalpies","partial_molar_entropies","partial_molar_int_energies","partial_molar_volumes","reference_pressure","thermal_expansion_coeff", "cp", "cv", "density", "h", "s", "g", "u", "v"]
+for name in getter_only_properties:
+    solution_properties = solution_properties + getter_only_properties_template.substitute(name=name, units=UNITS[name])
+
+setter_properties_template = Template("""
+    @property
+    def ${name}(self):
+        ${n0}, ${n1} = self._phase.${name}
+        return Q_(${n0}, ${u0}), Q_(${n1}, ${u1})
+    @${name}.setter
+    def ${name}(self, value):
+        ${n0} = value[0].to(${u0}).magnitude
+        ${n1} = value[1].to(${u1}).magnitude
+        self._phase.${name} = ${n0}, ${n1}
+""")
+
+setter_properties = ["TP", "DP","HP","SP","SV","TD","UV",]
+for name in setter_properties:
+    solution_properties = solution_properties + setter_properties_template.substitute(name=name, n0=name[0], u0=UNITS[name[0]], n1=name[1], u1=UNITS[name[1]])
+
+setter1_properties_template = Template("""
+    @property
+    def ${name}(self):
+        ${n0}, ${n1}, ${n2} = self._phase.${name}
+        return Q_(${n0}, ${u0}), Q_(${n1}, ${u1}), Q_(${n2}, ${u2})
+    @${name}.setter
+    def ${name}(self, value):
+        ${n0} = value[0].to(${u0}).magnitude
+        ${n1} = value[1].to(${u1}).magnitude
+        ${n2} = value[2].to(${u2}).magnitude
+        self._phase.${name} = ${n0}, ${n1}, ${n2}
+""")
+
+setter1_properties = ["TPX", "TPY", "DPX","DPY","HPX","HPY","SPX", "SPY","SVX","SVY","TDX","TDY","UVX","UVY"]
+for name in setter1_properties:
+    solution_properties = solution_properties + setter1_properties_template.substitute(name=name, n0=name[0], u0=UNITS[name[0]], n1=name[1], u1=UNITS[name[1]], n2=name[2], u2=UNITS[name[2]])
+
+pf_properties = ""
+
+pf_getter_only_properties_template = Template("""
+    @property
+    def ${name}(self):
+        return Q_(self._phase.${name},${units})
+""")
```

Comment:
This looks the same as the other `getter_only_properties_template`, can you reuse the earlier one?